conans/client/installer.py

```diff
@@ -576,7 +577,10 @@ def _propagate_info(self, node, using_build_profile):
 
         for n in node_order:
             if n not in transitive:
-                conan_file.output.info("Applying build-requirement: %s" % str(n.ref))
+                try:
+                    conan_file.output.info("Applying build-requirement: %s" % str(n.ref))
```

Comment:
?

Reply:
The printing of virtual consumers is disabled, because it was very confusing to users, seeing something like ``(virtual): Applying build-requirement...``. So ``conanfile.output.info`` or any other printing command will raise for ``RECIPE_VIRTUAL`` recipe.